compiler+runtime/CMakeLists.txt

```diff
@@ -176,8 +180,7 @@ endfunction()
 # can configure jank correctly during the build phase and then it'll just
 # continue to work with that setup once it's installed.
 execute_process(
-  COMMAND bash -c "${CMAKE_CXX_COMPILER} -xc++ -E -v /dev/null 2>&1 | sed -n -e '/^.include/,\${' -e '/^ \\/.*/p' -e '}' | sed 's/^\\s\\+/-I/' | sed 's/ (framework directory)//'"
-  RESULT_VARIABLE clang_system_includes_result
+  COMMAND bash -c "${CMAKE_CXX_COMPILER} -xc++ -E -v /dev/null 2>&1 | sed -n -e '/^.include/,\${' -e '/^ \\/.*/p' -e '}' | sed 's/^[[:space:]]\\{1,\\}/-I/' | sed 's/ (framework directory)//'"  RESULT_VARIABLE clang_system_includes_result
```

Comment:
Looks like a new line before `RESULT_VARIABLE` was accidentally removed.

Reply:
Also, in case it's not apparent why these changes were made, it's because the built-in `sed` on macOS is the BSD version not GNU and there are subtle differences in what expressions they support. `[[:space:]]` et al. are universal, so doing it this way supports BSD and GNU varieties of `sed`.

It's easy for this difference to be masked on macOS if you `brew install gnu-sed` and setup `sed` as an alias to `gsed`. I've bumped into that problem several times over the years.